pages/about-us.tsx

```diff
@@ -6,48 +6,118 @@ import styles from './about.module.scss'
 import Youtube from '../ui/youtube'
 import ColoredSection from '../ui/colored-section'
 import Polygons from '../ui/polygons'
-import kriszti from "../public/team/kriszti.png"
+
+import SpeakerPicture from '../ui/speaker-picture'
+
+import nec from '../public/team/nec.png'
+import gergo from '../public/team/gergo.png'
+import norbi from '../public/team/norbi.png'
+import ben from '../public/team/ben.png'
+import kriszti from '../public/team/kriszti.png'
+import dorka from '../public/team/dorka.png'
+import aniko from '../public/team/aniko.png'
+import freevo from '../public/team/freevo.png'
+import tim from '../public/team/tim.png'
 
 const team = [
   {
     name: 'Szabolcs Szabolcsi&#8209;Tóth',
-    image: 'https://nec.is/nec.png',
-    twitter: '_nec',
+    image: nec.src,
```

Comment:
in the implementation we are using fill + position relative parent, we can pass in simply the filename
(no need for static import then .src)